src/SFA.DAS.AdminService.Web/Views/Roatp/Apply/Gateway/ConfirmOutcomeApproved.cshtml

```diff
@@ -0,0 +1,71 @@
+@using SFA.DAS.AssessorService.Domain.Extensions;
+@using SFA.DAS.AssessorService.ApplyTypes.Roatp;
+
+@model SFA.DAS.AdminService.Web.ViewModels.Roatp.Gateway.RoatpGatewayConfirmOutcomeViewModel
+
+@{
+    ViewBag.Title = "Gateway - Overall gateway outcome";
+    Layout = "_Layout";
+}
+
+<div class="govuk-breadcrumbs">
+    <ol class="govuk-breadcrumbs__list">
+        <li class="govuk-breadcrumbs__list-item">
+            <a class="govuk-breadcrumbs__link" href="/Dashboard">RoATP dashboard</a>
```

Comment:
Same comment about hardcoded URLs that should be tag helpers

Reply:
Done